### DISCUSSION TITLE
Question on importing local mods

### DISCUSSION THREAD

**Discussion by luojiesi** [ebkr/r2modmanPlus#775] [Q&A]
Hi there, recently I started modding dyson sphere program. I have always used r2modman to manage my mods. So when I have built a mod, I tried to import it using r2modman's "import local mode" functionality. I clicked it from the menu and selected the .dll mod file. However nothing happened. I manually checked the data path that contains all my online mods, the local mod is not there. Anything I am doing wrong? Appreciate any help.

**Comment by MythicManiac** [comment]
@luojiesi check if https://github.com/ebkr/r2modmanPlus/wiki/Installing-mods-locally helps you
